### PR TITLE
Home focus

### PR DESCRIPTION
**Pull request by SRUJANA-PENUGONDA13**
### Added below features 

- User onboarding
- Display time, current weather in sync
- Wallpaper and quote changing on every refresh
- Main Focus for Today

Extension - 1
Extension - 2
Extension - 3

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+REACT_APP_WEATHER_API_HOST=community-open-weather-map.p.rapidapi.com
+REACT_APP_WEATHER_API_KEY=9d56960b7bmsh1ab16e59ddd16f8p11c6a7jsn4f9accfaf47e
```
Comment (viru2001) [inline .env]: remove this file ... don't reveal your API key

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    setImageURL(URL);
+  }, [windowWidth, windowHeight]);
+
+  const getWindowDimensions = () => {
+    const hasWindow = typeof window !== "undefined";
+    const width = hasWindow ? window.innerWidth : null;
+    const height = hasWindow ? window.innerHeight : null;
+    setWindowWidth(width.toString());
+    setWindowHeight(height.toString());
+  };
```
Comment (viru2001) [inline src/App.js]: you can make custom hook for getting window dimensions

File: src/components/Focus.js [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+const Focus = () => {
+  let [focus, setFocus] = useState("");
+  let [temperature, setTemperature] = useState("");
+  let [weatherIcon, setWeatherIcon] = useState("");
+  let [weatherStatus, setWeatherStatus] = useState("");
+  let [currentTime, setCurrentTime] = useState("");
+  let [greeting, setGreeting] = useState("");
+  let [displayStatus, setDisplayStatus] = useState("active");
+  let [focusUpdate, setFocusUpdate] = useState("");
+  let [quote, setQuote] = useState();
```
Comment (viru2001) [inline src/components/Focus.js]: since you have so many states here , you can use useReducer instead of useState

**Review by viru2001 — APPROVED** [review]
UI looks good.
Few suggestions - 
- you should make separate files for util functions
- never push your env file on github which contains API key

**Review by shrushti2000 — APPROVED** [review]
UI really looks good . All the points are covered in above review. Apart from this everything looks good to me.

File: .env (hunk 1 above)
Comment (SRUJANA-PENUGONDA13) [reply .env]: done

File: src/App.js (hunk 2 above)
Comment (SRUJANA-PENUGONDA13) [reply src/App.js]: done

File: src/components/Focus.js (hunk 3 above)
Comment (SRUJANA-PENUGONDA13) [reply src/components/Focus.js]: done

**Comment by SRUJANA-PENUGONDA13** [comment]
Thank You @viru2001 @shrushti2000 for reviewing

**Pull request merged**